community/people/sh/index.md

```diff
@@ -0,0 +1,25 @@
+---
+Layout: Person
```

Comment:
It is these labels, e.g. `layout` that need to be lower case

Reply:
Okay, I think that should do it :], hopefully. I didn't realize it the first time you mentioned it.